api/tm-catalog.openapi.yaml

```diff
@@ -53,6 +53,14 @@ paths:
           schema:
             type: string
           example: filter.mpn=B21,FC102
+        - name: 'filter.name'
+          in: query
+          description: |
+            Filters the inventory by TM name (consisting of 'author/manufacturer/mpn') having a prefix match.   
+            The filter works additive to other filters.
+          schema:
+            type: string
+          example: filter.name=darthvader/omnicorp/deathstar
```

Comment:
May be someone is a Star Trek fan ;-)
We should use something more neutral like `mycompany/abb/B21`

Reply:
Oh, come on! Who was ever against an unnecessary Star Wars or RoboCop reference in test/example data? Why so serious?